docs/source/recipes.rst

```diff
@@ -437,3 +437,38 @@ different environment.
 
    To test operations with a small interactive job, a 'test' group can be used
    to ensure that the operations do not try to run on multiple cores or GPUs.
+
+Pass options to operations run through containers or another environment
```

Comment:
Matched the -ing form of the other titles.


Could "in a container or other environment" be simplified to "in another process"? I'm not too familiar with the terminology.

```suggestion
Passing command line options to operations run in a container or other environment
```